examples/tutorials/Introduction to Langrangian Neural Networks.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<p>Spell check and make this part slightly more descriptive. Maybe show how you got the equations for r and theta if it's not too lengthy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3905/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>